packages/app/src/vis-packs/nexus/utils.ts

```diff
@@ -48,7 +48,8 @@ export function isNxGroup(entity: Entity): boolean {
     isGroup(entity) &&
     (isNxDataGroup(entity) ||
       hasAttribute(entity, 'default') ||
-      hasAttribute(entity, 'NX_class'))
+      getAttributeValue(entity, 'NX_class') === 'NXentry' ||
+      getAttributeValue(entity, 'NX_class') === 'NXroot')
```

Comment:
`['NXentry', 'NXroot'].includes()` doesn't work because the return type of `getAttributeValue` is `unknown`. I went with this instead of adding an assertion since we're most likely going to improve the type safety of `getAttributeValue` once we start fetching attribute values on demand.

Reply:
Yep, it is fine